Framework/API/inc/MantidAPI/MatrixWorkspace.h

```diff
@@ -364,7 +364,7 @@ class MANTID_API_DLL MatrixWorkspace : public IMDWorkspace,
   // Methods to set and access masked bins
   void maskBin(const size_t &workspaceIndex, const size_t &binIndex,
                const double &weight = 1.0);
-  void flagMasked(const size_t &spectrumIndex, const size_t &binIndex,
+  void flagMasked(const size_t &spectrumNum, const size_t &binIndex,
```

Comment:
I think this should still say `index`. It is the 0-based value for the vector lookup.